.github/workflows/rinkeby.yml

```
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: rinkeby deployment
+
+on:
+  push:
+    branches:
+      -  master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+    - name: yarn install
+      run: yarn install
+    - name: yarn lint
+      run: yarn run lint
```

Maybe we could use something more descriptive ?

```suggestion
    - name: Install dependencies
      run: yarn install
    - name: Run linter
      run: yarn run lint
```

---

ah, i think that like it is is ok because you have in the name the command that is going to be executed so is easier to see what is going on